src/actions/walletConnectActions.js

```diff
@@ -375,6 +377,23 @@ export const subscribeToWalletConnectConnectorEventsAction = (connector: WalletC
         return;
       }
 
+      if (method === ETH_SIGN_TYPED_DATA || method === ETH_SIGN_TYPED_DATA_V4) {
+        const { domain } = JSON.parse(params[1]);
```

Comment:
`JSON.parse` throws an error if there is an error in the JSON or something else like `params[index]` does not exist. Please ensure thta params[index] exists and / or wrap this in a try / catch and handle the error.

Reply:
Well, I made one more condition